task_schedule.cfg

```diff
@@ -0,0 +1,46 @@
+# Configuration file for task_schedule.pl to run astromon jobs
+
+subject           agasc supplement update
+timeout           86400             # Default tool timeout
+heartbeat_timeout 1                 # Maximum age of heartbeat file (seconds)
+iterations        1                 # Run once then shut down task_schedule
+print_error       1                 # Print full log of errors
+disable_alerts    0                 # Don't disable alerts since this jobs runs just once/day
+loud              0                 # Run loudly or quietly (production mode)
+
+# Data files and directories.  The *_dir vars can have $ENV{} vars which
+# get interpolated.  (Note lack of task name after TST_DATA because this is just for test).
+
+data_dir     /proj/sot/ska/data/agasc/rc/task_data       # Data file directory
+log_dir      /proj/sot/ska/data/agasc/rc/task_data  # Log file directory
+#bin_dir     ///     # Bin dir (optional, see task def'n)
+master_log   agasc_supplement_update.log             # Composite master log (created in log_dir)
+
+# Email addresses that receive an alert if there was a severe error in
+# running jobs (i.e. couldn't start jobs or couldn't open log file).
+# Processing errors *within* the jobs are caught with watch_cron_logs
+
+alert       javier.gonzalez@cfa.harvard.edu
+notify       javier.gonzalez@cfa.harvard.edu
+
+# Define task parameters
+#  cron: Job repetition specification ala crontab
+#  exec: Name of executable.  Can have $ENV{} vars which get interpolated.
+#        If bin_dir is defined then bin_dir is prepended to non-absolute exec names.
+#  log: Name of log.  Can have $ENV{} vars which get interpolated.
+#        If log is set to '' then no log file will be created
+#        If log is not defined it is set to <task_name>.log.
+#        If log_dir is defined then log_dir is prepended to non-absolute log names.
+#  timeout: Maximum time (seconds) for job before timing out
+
+# This has multiple jobs which get run in specified order
+# Note the syntax 'exec <number> : cmd', which means that the given command is
+# executed only once for each <number> of times the task is executed.
+
+<task agasc_supplement_update>
+      cron       * * * * *
+      check_cron * * * * *
+      exec 1: agasc-supplement-update --output-dir /proj/sot/ska/data/agasc/rc --report
+      exec 1: agasc-mag-estimate-report --output-dir /proj/sot/ska/data/agasc/rc
+      exec 1: cp -fr /proj/sot/ska/data/agasc/rc/supplement_reports/weekly/latest/stars/* /proj/sot/ska/data/agasc/rc/supplement_reports/stars
+</task>
```

Comment:
There should also be a `<check>` section something like:
```
      <check>
        <error>
          #    File            Expression
          #  -------------     ---------------------------
             acdc.log     error
             acdc.log     warning
             acdc.log     fatal
        </error>
      </check>
```
It looks like currently the logging is not being checked and split into `daily.?` directories. IIRC that gets done by `check_cron`.